### DISCUSSION TITLE
How can generate only models ?

### DISCUSSION THREAD

**Discussion by wildwind123** [sqlc-dev/sqlc#2608] [Q&A] [answered]
Hello. I want to use this soft for generate only models. How can i achieve it ?

**Comment by andrewmbenton** [comment, marked as answer]
I suppose the easiest way would be to put a dummy query (like `SELECT true;`) in your query file and then just ignore or discard the contents of `queries.sql.go`.

There is no configuration option currently to have `sqlc` only generate models.
